recipes/server.rb

```diff
@@ -80,6 +80,14 @@
     action :nothing
   end
 
+  bash 'Restore security context' do
+    user 'root'
+    code "/usr/sbin/restorecon -v #{node['mariadb']['mysqld']['default_datadir']}"
+    only_if '[ -f /usr/sbin/selinuxenabled ] && /usr/sbin/selinuxenabled'
```

Comment:
We should use the `selinux_enabled?` chef helper here instead.

This should be accessible if we add `Chef::Recipe.send(:include, Chef::Util::Selinux)` at the top of the recipe.